applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element_discontinuous.cpp

```diff
@@ -1174,6 +1232,26 @@ ModifiedShapeFunctions::Pointer GetContinuousShapeFunctionCalculator<3, 4>(
     return ModifiedShapeFunctions::Pointer(new Tetrahedra3D4ModifiedShapeFunctions(rElement.pGetGeometry(), rElementalDistances));
 }
 
+template <>
+ModifiedShapeFunctions::Pointer GetIncisedShapeFunctionCalculator<2, 3>(
+    const Element& rElement,
+    const Vector& rElementalDistancesWithExtrapolated,
+    const Vector& rElementalEdgeDistancesExtrapolated)
+{
+    return ModifiedShapeFunctions::Pointer(new Triangle2D3AusasIncisedShapeFunctions(rElement.pGetGeometry(),
+            rElementalDistancesWithExtrapolated, rElementalEdgeDistancesExtrapolated));
```

Comment:
```suggestion
    return Kratos::make_unique<Triangle2D3AusasIncisedShapeFunctions>(
        rElement.pGetGeometry(),
        rElementalDistancesWithExtrapolated,
        rElementalEdgeDistancesExtrapolated));
```
This should be possible as soon as we create the `DefineIncisedGeometryData` suggested above (eventually we should do so for the standard case).

(same comment applies for the 3D case below).

Reply:
I changed it for `AusasIncisedShapeFunctions` and `AusasModifiedShapeFunctions`